twitter_user.go

```diff
@@ -13,9 +13,11 @@ type User struct {
 	Following                      bool     `json:"following"`
 	FriendsCount                   int      `json:"friends_count"`
 	GeoEnabled                     bool     `json:"geo_enabled"`
+	HasExtendedProfile             bool     `json:"has_extended_profile"`
```

Comment:
Do you happen to have an example user for whom these two fields are enabled?


(I've been trying to move Anaconda towards the testing strategy that we use for a lot of other Go clients, but it's hard, because the Twitter API has a lot of optional fields that aren't always documented, and some are rather obscure.)